src/screens/Hooks/GroupACLHooks.res

```diff
@@ -31,20 +34,32 @@ let useUserGroupACLHook = () => {
 
   let fetchUserGroupACL = async () => {
     try {
-      let url = getURL(
-        ~entityName=USERS,
-        ~userType=#GET_GROUP_ACL,
-        ~methodType=Get,
-        ~queryParamerters=Some(`groups=true`),
-      )
+      let url = getURL(~entityName=USERS, ~userType=#GET_GROUP_ACL, ~methodType=Get)
       let response = await fetchDetails(url)
+      let dict = response->getDictFromJsonObject
+
       let groupsAccessValue =
-        response->getArrayFromJson([])->Array.map(ele => ele->JSON.Decode.string->Option.getOr(""))
-      setuserGroupACL(_ => Some(
-        groupsAccessValue
-        ->Array.map(ele => ele->mapStringToGroupAccessType)
-        ->convertValueToMap,
-      ))
+        getArrayFromDict(dict, "groups", [])->Array.map(ele =>
+          ele->JSON.Decode.string->Option.getOr("")
+        )
```

Comment:
Can directly use `getStrArrayFromDict` instead of `getArrayFromDict` . Then you donot need to do `Array.map` aprt

Reply:
Got it. Done